cmd/auth/describe.go

```diff
@@ -141,7 +141,10 @@ func render(ctx context.Context, cmd *cobra.Command, status *authStatus, templat
 		if err != nil {
 			return err
 		}
-		cmd.OutOrStdout().Write(buf)
+		_, err = cmd.OutOrStdout().Write(buf)
```

Comment:
Why don't we ignore error here but ignore in the similar cases below (cmd/auth/env.go for example)

Reply:
If function returns error, we propagate up (where it can still be ignored).